integration/system/event_test.go

```diff
@@ -58,3 +66,63 @@ func TestEvents(t *testing.T) {
 	}
 
 }
+
+func TestEventsEmptyOutput(t *testing.T) {
+	type apiResp struct {
+		resp *http.Response
+		err  error
+	}
+	chResp := make(chan *apiResp)
+	go func() {
+		resp, body, err := req.Get("/events")
+		body.Close()
+		chResp <- &apiResp{resp, err}
+	}()
+
+	select {
+	case r := <-chResp:
+		require.NoError(t, r.err)
+		assert.Equal(t, r.resp.StatusCode, http.StatusOK)
+	case <-time.After(3 * time.Second):
+		t.Fatal("timeout waiting for events api to respond, should have responded immediately")
+	}
+}
+
+func TestEventsBackwardsCompatible(t *testing.T) {
```

Comment:
Just `TestEvents` I think. "BaclwardsCompatible" doesn't add anything.

Reply:
@dnephin The test case was added in #18888 and switched from `JSONMessage` to `events.Message`. The backwards compatibility was maintained in the switch. I think it might make sense to keep the test. (For that reason we also could not use `client.Event` and have to use raw HTTP request).